satellite/satellitedb/dbx/satellitedb.dbx.sqlite3.sql

```diff
@@ -88,6 +88,7 @@ CREATE TABLE irreparabledbs (
 CREATE TABLE nodes (
 	id BLOB NOT NULL,
 	address TEXT NOT NULL,
+	last_ip TEXT NOT NULL,
```

Comment:
Shouldn't we get that information from the pointers rather than from here?

Reply:
No, the IPs can change quite frequently for DDNS Hosts @egonelbre

Reply:
Pointers store node id and we can get the address information from elsewhere. You know what actually. This shouldn't be stored in the database in the first place. Let's pull it into a separate struct... this would keep the load off the database.

Reply:
That wont work for uploads. For uploads we dont have any pointers to look at.

Reply:
Yeah, I agree. But, keeping it in-memory probably makes more sense anyways.

Reply:
I totally agree with that. the DB is way to overloaded with simple requests that are not critical

Reply:
So query for reputable nodes, check the returned node IDs against a map of lastIPs, throw out duplicates, exclude all nodeIDs with duplicate IPs from the next query and repeat?

Reply:
Actually, what's the best data structure to use for storing the lastIPs?

Reply:
Ah seems like I misunderstood what this `last_ip` meant, so it should be fine.

Reply:
As for storing the last_ip you can use []byte in the binary form... see https://golang.org/pkg/net/#IP

Reply:
Also postgres supports network types natively https://www.postgresql.org/docs/9.1/datatype-net-types.html, which probably would be faster than []byte/text

Reply:
Is this possible with DBX?

Reply:
@cam yes it believe it would be of type blob

Reply:
I mean the postgres type `inet`

Reply:
I'll do blob for now

Reply:
No dbx does not support inet

Reply:
@jenlij I think you may have the wrong Cam 😉